UnityProject/Assets/Scripts/Items/Weapons/Gun.cs

```diff
@@ -1105,23 +1097,15 @@ public enum WeaponType
 		Burst = 2
 	}
 
-#if UNITY_EDITOR
-	[CustomEditor(typeof(Gun), true)]
-	public class GunEditor : Editor
+	/// <summary>
+	///	//TODO: write a half decent summary
```

Comment:
to do it e.e

Reply:
shit i totally forgot about that, my bad